src/mscorlib/shared/System/IO/FileStream.Unix.cs

```diff
@@ -45,7 +45,7 @@ private SafeFileHandle OpenHandle(FileMode mode, FileShare share, FileOptions op
                 _asyncState = new AsyncState();
 
             // Translate the arguments into arguments for an open call.
-            Interop.Sys.OpenFlags openFlags = PreOpenConfigurationFromOptions(mode, _access, options); // FileShare currently ignored
+            Interop.Sys.OpenFlags openFlags = PreOpenConfigurationFromOptions(mode, _access, share, options); // FileShare currently ignored
```

Comment:
Please delete the `// FileShare currently ignored` comment... it's no longer ignored ;)